src/main/scala/view/scalaFX/components/tree/BunnyTreeView.scala

```diff
@@ -0,0 +1,71 @@
+package view.scalaFX.components.tree
+
+import model.Bunny
+import scalafx.scene.image.{Image, ImageView}
+import scalafx.scene.layout._
+import scalafx.scene.text.Text
+import view.scalaFX.components.tree.GenealogicalTreeView.{bunnyIconSize, spacingRegion}
+import view.scalaFX.components.tree.GenealogicalTreeViewConstants.{BUNNY_FONT_PROPORTION, BUNNY_INFO_PROPORTION}
+import view.scalaFX.utilities.Direction
+import view.scalaFX.utilities.Direction.Right
+import view.utilities.{BunnyImageUtils, ImageType}
+
+/**
+ * Represents the view on the Bunny in a tree.
+ */
+trait BunnyTreeView {
+  /** Reference to the model bunny entity */
+  val bunny: Bunny
+
+  /** Pane with actual view of the bunny */
+  val pane: Pane
+}
+
+object BunnyTreeView {
+  def apply(bunny: Bunny): BunnyTreeView = BunnyTreeViewImpl(bunny)
+
+  private case class BunnyTreeViewImpl(override val bunny: Bunny) extends BunnyTreeView {
+      override val pane = treeBunnyView(bunny)
+  }
+
+  private def bunnyView(bunny: Bunny): ImageView = new ImageView {
+    image = BunnyImageUtils.bunnyToImage(bunny, ImageType.Normal)
+    fitWidth = bunnyIconSize
+    fitHeight = bunnyIconSize
+    preserveRatio = true
+    scaleX = Direction.scaleXValue(Right)
+  }
+
+  private def deadImageView: ImageView = new ImageView {
+    image = new Image("/img/death.png")
+    fitWidth = bunnyIconSize/BUNNY_INFO_PROPORTION
+    fitHeight = bunnyIconSize/BUNNY_INFO_PROPORTION
+  }
+
+  private def mutationImageView: ImageView = new ImageView {
+    image = new Image("/img/mutation.png")
+    fitWidth = bunnyIconSize/BUNNY_INFO_PROPORTION
+    fitHeight = bunnyIconSize/BUNNY_INFO_PROPORTION
+  }
```

Comment:
Questi due metodi sono praticamente uguali cambia solo il percorso dell'immagine, quindi si potrebbe fare un metodo che prende il path in input e restituisce l'immagine in questo modo eviti la ripetizione di codice